modules/serverless-ossm-setup.adoc

```diff
@@ -177,6 +178,172 @@ spec:
+include::snippets/technology-preview.adoc[leveloffset=+4]
+
+. Apply the `KnativeEventing` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Install Knative Kafka by creating the following `KnativeKafka` custom resource definition (CRD), which also enables the Istio integration:
++
+[source,yaml]
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+  name: knative-kafka
+  namespace: knative-eventing
+spec:
+  channel:
+    enabled: true
+    bootstrapServers: <bootstrap_servers> <1>
+  source:
+    enabled: true
+  broker:
+    enabled: true
+    defaultConfig:
+      bootstrapServers: <bootstrap_servers> <1>
+      numPartitions: <num_partitions>
+      replicationFactor: <replication_factor>
+  sink:
+      enabled: true
+  workloads: <2>
+  - name: kafka-controller
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-broker-receiver
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-broker-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-channel-receiver
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-channel-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-source-dispatcher
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: kafka-sink-receiver
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+----
+<1> The Apache Kafka cluster URL, for example: `my-cluster-kafka-bootstrap.kafka:9092`.
+<2> Enables sidecar injection for Knative Kafka pods.
+
+:FeatureName: The Knative Eventing integration with {SMProductShortName}
+include::snippets/technology-preview.adoc[leveloffset=+4]
+
+. Apply the `KnativeKafka` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Install `ServiceEntry` to make {SMProductName} aware of the communication between `KnativeKafka` components and an Apache Kafka cluster:
++
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: kafka-cluster
+  namespace: knative-eventing
+spec:
+  hosts: <1>
+    - <bootstrap_servers_without_port>
+  exportTo:
+    - "."
+  ports: <2>
+    - number: 9092
+      name: tcp-plain
+      protocol: TCP
+    - number: 9093
+      name: tcp-tls
+      protocol: TCP
+    - number: 9094
+      name: tcp-sasl-tls
+      protocol: TCP
+    - number: 9095
+      name: tcp-sasl-tls
+      protocol: TCP
+    - number: 9096
+      name: tcp-tls
```

Comment:
same problem as above, this name is already used above, so I suggest something like
```suggestion
      name: tcp-tls-noauth
```

Reply:
(alternatively, if we do not even want to hint at using non-tls ports, we could just remove those ports althogether